Planner/src/TaskDAO.java

```diff
@@ -0,0 +1,219 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+import javax.swing.DefaultComboBoxModel;
+
+public class TaskDAO {
+	
+	static ArrayList<Task> getTasks(ArrayList<User> users) {
+		ArrayList<Task> tasks = new ArrayList<Task>();
+		
+		try {
+			// Step 1: Allocate a database 'Connection' object
+			Connection conn = DriverManager.getConnection(
+				"jdbc:mysql://localhost:3306/planner?useSSL=false", "Justyna", "qwer1234");
+				// MySQL: "jdbc:mysql://hostname:port/databaseName", "username", "password"
+	 
+	        // Step 2: Allocate a 'Statement' object in the Connection
+			Statement stmt = conn.createStatement();
+			
+			// Step 3: Execute a SQL SELECT query, the query result
+			//  is returned in a 'ResultSet' object.
+			String strSelect = "select taskID, nameTask, descriptionTask, assignedTo, startTime, endTime from tasks";
+			System.out.println("The SQL query is: " + strSelect); // Echo For debugging
+	        System.out.println();
+	 
+	        ResultSet rset = stmt.executeQuery(strSelect);
+	 
+	        // Step 4: Process the ResultSet by scrolling the cursor forward via next().
+	        //  For each row, retrieve the contents of the cells with getXxx(columnName).
+	        System.out.println("The records selected are:");
+	        int rowCount = 0;
+	        while(rset.next()) {   // Move the cursor to the next row, return false if no more row
+	        	String nameTask = rset.getString("nameTask");
+	            String descriptionTask = rset.getString("descriptionTask");
+	            Date startTime = null;
+	            Date endTime = null;
+	            try {
+	            	startTime = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(rset.getString("startTime"));
+	            	endTime = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(rset.getString("endTime"));
+	            } catch (ParseException e) {
+	            	System.out.println("ParseException date");
+	            }
+	            int assignedTo = rset.getInt("assignedTo");
+	            int taskID = rset.getInt("taskID");
+	            /*
+	            // Step 3 & 4: Issue a SELECT to check the UPDATE.
```

Comment:
Here, you need to replace this commented code with a for loop in the users array. You need to search the user that has the id == assignedTo and put this user in the constructor at line 65. Do the same thing for insert and update methods.